InformationScripting/src/queries/UnionOperator.cpp

```diff
@@ -28,9 +28,9 @@
 
 namespace InformationScripting {
 
-QList<TupleSet> UnionOperator::execute(QList<TupleSet> input)
+QList<Optional<TupleSet> > UnionOperator::execute(QList<TupleSet> input)
 {
-	if (input.size() <= 1) return input;
+	if (input.size() <= 1) return {{"Union requires 2 inputs"}};
```

Comment:
Hmm, now that I look at this example, I'm not sure if we should have `QList<Optional<TupleSet>>` or `Optional<QList<TupleSet>>`. What are pros and cons of each approach?